### ISSUE TITLE
Renesa Plus Fan Speed

### ISSUE THREAD

**Issue by Zoro-6191** [dasshubham762/atomberg-integration#27]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

I can't change fan speed from main UI without long pressing.

Its also has a rather small area where i can long press, I often miss it.

### Describe the solution you'd like

I'd really like sort of a knob or just a step regulator with 6 speeds ( 0 to max ) on the main page for the ease.

### Describe alternatives you've considered

Pretty straight forward. Instead of a On/Off button, I propose a step regulator type UI, or even a combination of both.

### Additional context

None

**Comment by Zoro-6191** [comment]
I wrote long press, I meant just a Tap

**Comment by sphoenixp** [comment]
for me On Off is not working rest everything is working. Erica

**Comment by aic4u** [comment]
Yes, same here - started getting error : **_fan does not support action fan.turn_on_**.  But speed setting helps to turn it on.

**Comment by ArunSharon** [comment]
I have no option for speed control via HA. My model is Aris Starlight. Is there any additional config needed to enable speed control options.

**Comment by cateyeddd** [comment]
Erica smart fan is working with WIFI & with Amazon Alexa but the entities in Home Assistant is greyed out. Not working. Tried generating new APIs and adding it to Home Assistant.

**Comment by Zoro-6191** [comment]
> Hey guys can someone please tell me how we can control the fan speed ? All i see on dashboard is Fan Light On/Off and Fan Power on/off

click on the icon, that should pop the slider up

**Comment by dasshubham762** [comment]
Closing the issue as the Speed Control slider is already available.

**Comment by cateyeddd** [comment]
Hey @dasshubham762 havent heard from you. The problem still remains.

**Comment by dasshubham762** [comment]
Hi @cateyeddd do you have the fan and the HA instance on separate VLAN?

**Comment by cateyeddd** [comment]
> Hi @cateyeddd do you have the fan and the HA instance on separate VLAN?

Yes. Fan on IOT VLAN & HA on Main VLAN

**Comment by dasshubham762** [comment]
> Yes. Fan on IOT VLAN & HA on Main VLAN

In this case, there are two possible solutions:
1. You can connect the fan to the same network that Home Assistant connected to.
2. Relay the UDP broadcasts on port 5625 from IOT VLAN to Main VLAN. (This is the setup I am currently using.)

**Comment by cateyeddd** [comment]
> > Yes. Fan on IOT VLAN & HA on Main VLAN
> 
> In this case, there are two possible solutions:
> 1. You can connect the fan to the same network that Home Assistant connected to.
> 2. Relay the UDP broadcasts on port 5625 from IOT VLAN to Main VLAN. (This is the setup I am currently using.)

Did try to relay the UDP broadcasts on port 5625 from IOT to Main VLAN. But it didn't work

**Comment by dasshubham762** [comment]
> Did try to relay the UDP broadcasts on port 5625 from IOT to Main VLAN. But it didn't work

Forgot to mention, you also need add a firewall rule to allow UDP traffic on port 5625 from IOT to Main VLAN. If already added then let me know how did you relay the broadcasts.

**Comment by cateyeddd** [comment]
> add

I already had a firewall rule to allow port 5625. But the problem was of the different VLANS. I moved Atomberg to my main VLAN & now it's working. But I prefer to isolate my IOT network & my main Network. If possible, kindly add support for this. Thank You!
